Validator/Constraints/HasAllowedFileExtensionValidator.php

```diff
@@ -2,11 +2,9 @@
 
 namespace Lch\MediaBundle\Validator\Constraints;
 
-use Lch\MediaBundle\Behavior\Storable;
```

Comment:
Why did this trait was removed?

Reply:
Sorry, i removed all unused classes, so i guess i remove with those classes.
I will return it